MathLib/LinAlg/UnifiedMatrixSetters.cpp

```diff
@@ -83,25 +83,35 @@ double norm(PETScVector const& x)
 void setVector(PETScVector& v,
                std::initializer_list<double> values)
 {
-    (void) v; (void) values;
-    // TODO implement
-}
+    std::vector<double> const vals(values);
+    std::vector<PETScVector::IndexType> idcs(vals.size());
+    std::iota(idcs.begin(), idcs.end(), 0);
 
+    v.set(idcs, vals);
+}
 
 void setMatrix(PETScMatrix& m,
                PETScMatrix::IndexType const rows,
                PETScMatrix::IndexType const cols,
                std::initializer_list<double> values)
 {
-    (void) m; (void) rows; (void) cols; (void) values;
-    // TODO implement
+    m.setZero();
+    addToMatrix(m, rows, cols, values);
```

Comment:
> There is no check if it fits the matrix dimensions, though. Do we have to check that?

Is it possible that the values for `rows` and `cols` can be set independently from the matrix dimensions? How big is the potential to misuse the interface?

> There is an assertion in addToMatrix(). However, that's disabled in release mode. Does that suffice?

Do you expect enabling the check in release mode will be run time critical? If not, I would have a better feeling with enable check.